packages/latte-phpstan-compiler/src/PhpParser/NodeVisitor/NClassNodeVisitor.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\LattePHPStanCompiler\PhpParser\NodeVisitor;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\BinaryOp\Concat;
+use PhpParser\Node\Expr\Ternary;
+use PhpParser\Node\Scalar\String_;
+use PhpParser\Node\Stmt\Echo_;
+use PhpParser\Node\Stmt\Expression;
+use PhpParser\NodeVisitorAbstract;
+
+/**
+ * from: <code> echo ($ʟ_tmp = \array_filter(['class1', $var ? 'class2' : \null])) ? ' class="' .
+ * \Latte\Runtime\Filters::escapeHtmlAttr(\implode(" ", \array_unique($ʟ_tmp))) . '"' : ""; </code>
+ *
+ * to: <code> $ʟ_tmp = \array_filter(['class1', $var ? 'class2' : \null]); echo ' class="' .
+ * \Latte\Runtime\Filters::escapeHtmlAttr(\implode(" ", \array_unique($ʟ_tmp))) . '"'; </code>
```

Comment:
This still looks quite confusing and hard to read, what is going on. The PHPStan needs to know only that `$var` is a string.

What about making it PHPSta and human-readable?

```diff
-echo ($ʟ_tmp = \array_filter(['class1', $var ? 'class2' : \null])) ? ' class="' .
-\Latte\Runtime\Filters::escapeHtmlAttr(\implode(" ", \array_unique($ʟ_tmp))) . '"' : "";
+echo ' class="' : $var . '"'
```

Reply:
Rewritten, I think we need ad leasy implode. Check it now

Reply:
Very nice, now it looks clean and easy to read :+1:  Thank you